### DISCUSSION TITLE
Is there a way to set some params?

### DISCUSSION THREAD

**Discussion by mreismendes** [winedarksea/AutoTS#67] [Q&A]
Hello there!

In my specific problem, I would like to train the data using 4 specifc lengths of `"context_length"`, i.e, [12,24,36,48].

Is it possible?

Thank you

**Comment by winedarksea** [comment]
@mreismendes 

Short answer: no. 
Also, 'context_slicer' has been replaced by the 'Slice' transformer.

There's a bit of a hack to do this, though. 
1. Train your model as usual
2. Export the model template for however many models you want
3. Open the model template in a text editor
4. Manually edit the templates to include the Slice transformer with the method=N parameter, where N is either an integer (like above, 12, 24...) or a decimal of the full data to use (0.5 = use 50% of history).
5. Run modeling again, this time importing the template. Remember to set method='only' on import and generations=0 if you don't want it to start generating more new models which may not include Slice as desired.

**Reply by winedarksea** [reply]
There's another sorta helpful thing in the latest version, where you can pass `transformer_list=['Slice']` where you can make sure that Slice is the only transformation used. Or perhaps more usefully, you can pass in a dictionary transformer_list up-weighting that Transformer's occurrence. This doesn't help you change the params, but it will make it occur in most models.
```python
transformer_list= {
    "Slice": 0.9,
    'MinMaxScaler': 0.05,
    'PowerTransformer': 0.1,
    'QuantileTransformer': 0.1,
    'MaxAbsScaler': 0.05,
    'StandardScaler': 0.04,
    'RobustScaler': 0.05,
    'Detrend': 0.05,
    'RollingMeanTransformer': 0.02,
    'DifferencedTransformer': 0.1,
    'SinTrend': 0.01,
    'PctChangeTransformer': 0.01,
    'CumSumTransformer': 0.02,
    'PositiveShift': 0.02,
    'Log': 0.01,
    'SeasonalDifference': 0.08,
    'cffilter': 0.01,
    'bkfilter': 0.05,
    "ClipOutliers": 0.05,
    "Discretize": 0.05,
    "CenterLastValue": 0.01,
    "Round": 0.05,
}
